diff --git a/node_modules/@mdx-deck/gatsby-plugin/src/keyboard.js b/node_modules/@mdx-deck/gatsby-plugin/src/keyboard.js
index 4c65c76..7617ebc 100644
--- a/node_modules/@mdx-deck/gatsby-plugin/src/keyboard.js
+++ b/node_modules/@mdx-deck/gatsby-plugin/src/keyboard.js
@@ -16,6 +16,8 @@ const keys = {
   pageDown: 34,
 }
 
+const inputElements = ['input', 'select', 'textarea', 'a', 'button']
+
 export const useKeyboard = () => {
   const context = useDeck()
 
@@ -24,6 +26,10 @@ export const useKeyboard = () => {
       if (e.metaKey) return
       if (e.ctrlKey) return
 
+      // ignore custom keyboard shortcuts when elements are focused
+      const el = document.activeElement.tagName.toLowerCase()
+      if (inputElements.includes(el)) return
+
       if (e.altKey) {
         switch (e.keyCode) {
           case keys.p:
